site/docs/pages/wallet/wallet-dropdown-disconnect.mdx

```diff
@@ -0,0 +1,83 @@
+import {
+  Address,
+  Avatar,
+  Name,
+  Identity,
+  EthBalance,
+} from '@coinbase/onchainkit/identity';
+import { color } from '@coinbase/onchainkit/theme';
+import { 
+  ConnectWallet,
+  Wallet,
+  WalletDropdown,
+  WalletDropdownLink,
+  WalletDropdownBaseName,
+  WalletDropdownDisconnect,
+} from '@coinbase/onchainkit/wallet';
+import WalletComponents from '../../components/WalletComponents';
+
+# `<WalletDropdownDisconnect />`
+
+The `WalletDropdownDisconnect` component is used to disconnect the wallet from the application.
+
+## Usage 
```

Comment:
For every component is important to teach each props, how to use it.

So in this case we should teach how to use:
- className
- text